.github/workflows/E2ETest.yml

```
@@ -69,7 +69,7 @@ jobs:
       - name: Setup E2E tests
         shell: pwsh
         run: |
-          $env:MSSQL_SA_PASSWORD = "$([Guid]::NewGuid())"
+          $env:MSSQL_SA_PASSWORD = "Str0ngP@ss!2025"
```

The GUID doesn't guarantee that the generated output meets the SQL password requirements every time. If it doesn't, the db will automatically shut down and thus cause the tests failed. I updated it to be like this, and it worked good with my local end. But I still see `sa log in failed` failures in the CI logs. Could you check this?@andystaples

---

Hardcoded passwords are likely to be flagged. How about something like this?
```suggestion
          $env:MSSQL_SA_PASSWORD = "TEST12_$([Guid]::NewGuid())!"
```